doc/tutorials/introduction/openvino_install/openvino_install.markdown

```diff
@@ -0,0 +1,547 @@
+    @endcode
+
+3.  Compile and install OpenCV:
+    @code{.bash}
+    cmake -G Ninja \
+          -D BUILD_INFO_SKIP_EXTRA_MODULES=ON \
+          -D BUILD_EXAMPLES=OFF \
+          -D BUILD_JASPER=OFF \
+          -D BUILD_JAVA=OFF \
+          -D BUILD_JPEG=ON \
+          -D BUILD_APPS_LIST=version \
+          -D BUILD_opencv_apps=ON \
+          -D BUILD_opencv_java=OFF \
+          -D BUILD_OPENEXR=OFF \
+          -D BUILD_PNG=ON \
+          -D BUILD_TBB=OFF \
+          -D BUILD_WEBP=OFF \
+          -D BUILD_ZLIB=ON \
+          -D WITH_1394=OFF \
+          -D WITH_CUDA=OFF \
+          -D WITH_EIGEN=OFF \
+          -D WITH_GPHOTO2=OFF \
+          -D WITH_GSTREAMER=ON \
+          -D OPENCV_GAPI_GSTREAMER=OFF \
+          -D WITH_GTK_2_X=OFF \
+          -D WITH_IPP=ON \
+          -D WITH_JASPER=OFF \
+          -D WITH_LAPACK=OFF \
+          -D WITH_MATLAB=OFF \
+          -D WITH_MFX=ON \
+          -D WITH_OPENCLAMDBLAS=OFF \
+          -D WITH_OPENCLAMDFFT=OFF \
+          -D WITH_OPENEXR=OFF \
+          -D WITH_OPENJPEG=OFF \
+          -D WITH_QUIRC=OFF \
+          -D WITH_TBB=OFF \
+          -D WITH_TIFF=OFF \
+          -D WITH_VTK=OFF \
+          -D WITH_WEBP=OFF \
+          -D CMAKE_USE_RELATIVE_PATHS=ON \
+          -D CMAKE_SKIP_INSTALL_RPATH=ON \
+          -D ENABLE_BUILD_HARDENING=ON \
+          -D ENABLE_CONFIG_VERIFICATION=ON \
+          -D ENABLE_PRECOMPILED_HEADERS=OFF \
+          -D ENABLE_CXX11=ON \
+          -D INSTALL_PDB=ON \
+          -D INSTALL_TESTS=ON \
+          -D INSTALL_C_EXAMPLES=ON \
+          -D INSTALL_PYTHON_EXAMPLES=ON \
+          -D CMAKE_INSTALL_PREFIX=install \
+          -D OPENCV_SKIP_PKGCONFIG_GENERATION=ON \
+          -D OPENCV_SKIP_PYTHON_LOADER=OFF \
+          -D OPENCV_SKIP_CMAKE_ROOT_CONFIG=ON \
+          -D OPENCV_GENERATE_SETUPVARS=OFF \
+          -D OPENCV_BIN_INSTALL_PATH=bin \
+          -D OPENCV_INCLUDE_INSTALL_PATH=include \
+          -D OPENCV_LIB_INSTALL_PATH=lib \
+          -D OPENCV_CONFIG_INSTALL_PATH=cmake \
+          -D OPENCV_3P_LIB_INSTALL_PATH=3rdparty \
+          -D OPENCV_SAMPLES_SRC_INSTALL_PATH=samples \
+          -D OPENCV_DOC_INSTALL_PATH=doc \
+          -D OPENCV_OTHER_INSTALL_PATH=etc \
+          -D OPENCV_LICENSES_INSTALL_PATH=etc/licenses \
+          -D OPENCV_INSTALL_FFMPEG_DOWNLOAD_SCRIPT=ON \
+          -D BUILD_opencv_world=OFF \
+          -D BUILD_opencv_python2=OFF \
+          -D BUILD_opencv_python3=ON \
+          -D PYTHON3_PACKAGES_PATH=install/python/python3 \
+          -D PYTHON3_LIMITED_API=ON \
+          -D HIGHGUI_PLUGIN_LIST=all \
+          -D OPENCV_PYTHON_INSTALL_PATH=python \
+          -D CPU_BASELINE=SSE4_2 \
+          -D OPENCV_IPP_GAUSSIAN_BLUR=ON \
+          -D WITH_INF_ENGINE=ON \
+          -D InferenceEngine_DIR=<OpenVINO_ROOT_DIRECTORY>/runtime/cmake/ \
+          -D ngraph_DIR=<OpenVINO_ROOT_DIRECTORY>/runtime/cmake/ \
+          -D INF_ENGINE_RELEASE=2022010000 \
```

Comment:
There is existing practice of not posting build instruction in the **fixed** documentation (no way to change content after release). Documentation should be targeted to API related stuff.
We post such build instructions on Wiki instead (especially if instruction depends on external libraries).

Ref:
1. There is already OpenVINO specific page: https://github.com/opencv/opencv/wiki/Intel%27s-Deep-Learning-Inference-Engine-backend
2. Tengine: https://github.com/opencv/opencv/wiki/Tengine-based-acceleration
3. TIM-VX: https://github.com/opencv/opencv/wiki/TIM-VX-Backend-For-Running-OpenCV-On-NPU